core/modules/layout/plugins/access/entity_bundle_layout_access.inc

```diff
@@ -4,6 +4,11 @@
  * Plugin to provide access control based upon entity bundle.
  */
 class EntityBundleLayoutAccess extends LayoutAccess {
+  /**
```

Comment:
There should be a blank line between the start of the class and the property/method definition. See https://docs.backdropcms.org/php-standards#object-oriented-code. There are a few places in this PR where this needs to be fixed.

Reply:
@herbdool right, there should be. I think, I fixed them all now. :+1: